File: backend/app/routers/inspiration.py
from fastapi import APIRouter

from app.schemas.inspiration import InspirationResponse

router = APIRouter()


@router.get("/today", response_model=InspirationResponse)
async def get_today_inspiration():
    """Get today's inspiration bundle (quote + fact + activity)."""
    # TODO: Call inspiration_service
    return InspirationResponse()


@router.post("/refresh", response_model=InspirationResponse)
async def refresh_inspiration():
    """Get a new set of inspiration content."""
    # TODO: Call inspiration_service.refresh()
    return InspirationResponse()

File: frontend/src/api/inspiration.ts
import apiClient from "./client";
import type { Inspiration } from "../types/briefing";

export async function fetchTodayInspiration(): Promise<Inspiration> {
  const { data } = await apiClient.get<Inspiration>("/inspiration/today");
  return data;
}

export async function refreshInspiration(): Promise<Inspiration> {
  const { data } = await apiClient.post<Inspiration>("/inspiration/refresh");
  return data;
}
